ts/Maps/MapPointer.ts

```diff
@@ -160,6 +169,12 @@ extend<Pointer|Highcharts.MapPointer>(Pointer.prototype, {
                 Math.abs(delta) < 1 ? false : void 0
             );
         }
+
+        // Set a timeout to reset the animation flag after the animation
+        // finishes
+        setTimeout(function (): void {
+            chart.isAnimating = false;
+        }, 50);
```

Comment:
When zooming with the trackpad, it is much better than it was, but it is still a jerky movement. It's not super smooth like for example Google Maps. I tried to lower the timeout to 25 milliseconds. Then the animation is much smoother, but if we zoom quickly (with the trackpad), it sometimes jumps around. Could you have a look at this?

Reply:
Yes, this is the limitation with the `isAnimating` flag that I also came across and discussed in a private message. I'll try to experiment with a different approach, but here what worked was temporarily disabling or blocking the animation for these fast non-standard movements.